plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/TransactionLogUtil.java

```diff
@@ -42,8 +41,7 @@ public static Map<String, Optional<String>> canonicalizePartitionValues(Map<Stri
     {
         return partitionValues.entrySet().stream()
                 .collect(toImmutableMap(
-                        // canonicalize partition keys to lowercase so they match column names used in DeltaLakeColumnHandle
-                        entry -> canonicalizeColumnName(entry.getKey()),
+                        Map.Entry::getKey,
```

Comment:
Is this method still needed?
Same question for the field `AddFileEntry#canonicalPartitionValues` .

We don't have any canonicalization anymore here.

Reply:
It's still needed because the value is still canonicalized in this method.